recipe/run_test.sh

```diff
@@ -52,13 +54,12 @@ numba -s
 python -m numba.tests.test_runtests
 
 if [[ "$archstr" == 'aarch64' ]] || [[ "$archstr" == "ppc64le" ]]; then
-	echo 'Running only a slice of tests'
-	$SEGVCATCH python -m numba.runtests -b -j --random='0.15' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
+	echo "Skipping numba test suite on $archstr"
+	#echo 'Running only a random selection of tests'
+	#$SEGVCATCH python -m numba.runtests -b --random='0.15' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
```

Comment:
The `ppc64le` builds have been hanging in the test suite for a few releases now ( https://github.com/conda-forge/numba-feedstock/issues/102 ). It may just be too much for Travis CI to handle. So skipping those.

Also `aarch64` needs some patches to LLVM to fix some test issues ( https://github.com/conda-forge/llvmdev-feedstock/pull/192 ). This is in review. Once new patched LLVM packages are shipped, this should be fixed.

Can open a PR to re-enable the skipped tests after. Then merge once the issues are resolved in the LLVM package. Since we use the LLVM dynamic library, no rebuilding of Numba should be needed to get those fixes.

Reply:
Submitted PR ( https://github.com/conda-forge/numba-feedstock/pull/119 ) to try reverting